pkg/kubelet/kuberuntime/fake_kuberuntime.go

```diff
@@ -0,0 +1,482 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kuberuntime
```

Comment:
So move the tests that need them too :)

We could also rename them to `fake_..._test.go` to not have them be compiled in so long as they won't be used from any other packages.

I know the fake kubelet is called from non-test code and is part of kubelet.go so we have a precedent for this, but imo it's a mistake and we should do *something* to make it so that doesn't happen here.